docs/multicluster/upgrade.md

```diff
@@ -0,0 +1,49 @@
+# Antrea Multi-cluster Upgrade Guide
```

Comment:
Consider to split long lines to 80 or 100 or 120 letters wide. It is easier to edit and review.

Reply:
done